.rubocop.yml

```diff
@@ -0,0 +1,1158 @@
+  AllCops:
```

Comment:
While the template was auto-generated, I created it with Code Climate and merged in anything that was customised in the previous file. For example, the default is to preference single-quoted strings but I've enforced double-quotes.

There are Rubocop validation errors but they are all related to the use of single-quotes.